### ISSUE TITLE
Fix UI issues related to MV3

### ISSUE THREAD

**Issue by chunyuema** [privacy-tech-lab/gpc-privacy-choice#170]
This issue will attempt to fix any broken UI after the migration to MV3: 

- [ ] "Review or modify settings" button no longer works #168 mentioned by @bella-tassone

**Comment by bella-tassone** [comment]
I seemed to resolve the "Modify Settings" bug by inserting the highlighted code below in <background.js>. Need to test some more before I confirm it fixed, but wanted to document the chang
<img width="646" alt="Screen Shot 2022-02-01 at 3 41 01 PM" src="https://user-images.githubusercontent.com/54873610/152048133-f48d907e-7890-4729-9743-e4d049f75f54.png">
e.

**Comment by chunyuema** [comment]
> I seemed to resolve the "Modify Settings" bug by inserting the highlighted code below in `background.js`. Need to test some more before I confirm it fixed, but wanted to document the change.

Great! Logically I can see how this can solve the problem too :)

**Comment by bella-tassone** [comment]
A question I came upon while testing different schemes: I noticed that for scheme 5, we don't record in the firebase whatever privacy profile they end up with. We record the decisions they make (the five choices before the algorithm makes it's choice), but nothing more. Should we change this? We could easily enough do the math ourselves to see what they end up with, but I think it would make sense to record what privacy profile the scheme ends up assigning. Thoughts? @chunyuema @ekuller @SebastianZimmeck

**Comment by chunyuema** [comment]
Can you provide more details @bella-tassone ? I think once the learning phase is done, the privacy profile is recorded as the initial privacy profile setting.

**Comment by bella-tassone** [comment]
> Can you provide more details @bella-tassone ? I think once the learning phase is done, the privacy profile is recorded as the initial privacy profile setting.

My mistake! You are correct :)

**Comment by bella-tassone** [comment]
Issue 170 has successfully been merged into main, so I will be closing this issue.

**Comment by bella-tassone** [comment]
Issue 170 has successfully been merged into main, so I will be closing this issue.
